aptos-move/aptos-vm/src/move_vm_ext/respawned_session.rs

```diff
@@ -155,6 +157,36 @@ impl<'r> TResourceView for ExecutorViewWithChangeSet<'r> {
     }
 }
 
+impl<'r> TResourceGroupView for ExecutorViewWithChangeSet<'r> {
+    type Key = StateKey;
+    type Layout = MoveTypeLayout;
+    type Tag = StructTag;
+
+    fn resource_group_size(&self, _state_key: &Self::Key) -> anyhow::Result<u64> {
+        // In respawned session, gas is irrelevant, so we return 0 (GroupSizeKind::None).
```

Comment:
even it doesn't matter this feels weird, we should be able to provide the correct value?

Reply:
0 can be correct behavior, e.g. now ResourceGroupAdapter also returns Ok(0) if GroupSizeKind is None (super old gas versions :).

The issue of returning actually correct thing is it would require applying any changes in VMChangeSet and that's not trivial, while it's also unnecessary. Would do unimplemented, but the ignoring of gas may happen somewhere in the callers and wouldn't want to panic.